apps/code-examples/src/app/code-examples/datetime/date-range-picker/custom-calculator/demo.component.html

```diff
@@ -19,6 +17,9 @@
       }
     </sky-date-range-picker>
   </form>
+  <p>
+    Selected calculator: {{ selectedCalculator()?.shortDescription$ | async }}
```

Comment:
Demonstration for how to print a custom calculator's short description in the template.